src/azure-cli-core/azure/cli/core/commands/arm.py

```diff
@@ -187,6 +187,52 @@ def resource_exists(cli_ctx, subscription, resource_group, name, namespace, type
     return existing
 
 
+def register_global_policy_argument(cli_ctx):
+
+    def add_global_policy_argument(_, **kwargs):
+        command_table = kwargs.get('commands_loader').command_table
+
+        if not command_table:
+            return
+
+        class ChangeReferenceAction(argparse.Action):  # pylint:disable=too-few-public-methods
+
+            def __call__(self, parser, namespace, value, option_string=None):
+                # save change reference to CLI context
+                cmd = getattr(namespace, 'cmd', None) or getattr(namespace, '_cmd', None)
```

Comment:
I guess this line is from

https://github.com/Azure/azure-cli/blob/4a15044c9d14c0ea7d207f5defbecffec19831bd/src/azure-cli-core/azure/cli/core/commands/__init__.py#L1521

Why can there be both `cmd` and `_cmd`? Shouldn't there be only one?